Updated introductory process timeline to new proposal (#202)
ComputerScienceHouse/Constitution: constitution.tex

\asubsubsection{The Introductory Process}
The Introductory Process is designed to provide an easy means for Introductory Members to meet existing House members, learn House history, demonstrate their involvement potential to House, and allow existing House members to evaluate them for Active Membership.
\asubsubsubsection{The Evaluation Period}
The Process will begin between the first and second week of each semester, then last six (6) weeks, starting when the Evaluations Director initiates it.
The Executive Board may hold a closed ballot Simple Majority vote to extend the Introductory Process or the Introductory Packet.
If an Introductory Process is extended, the Introductory Evaluation shall occur at the termination of the extended Process, which may be outside of the period defined in \ref{Introductory Evaluation}.

> The Introductory Process may be held twice a semester at the discretion of the evaluations director. If the evaluations director deems it necessary, the second intro process must begin within a week of the first process ending, and must last the same six (6) weeks.

Reviewer: `within a week of the first process ending` when is this? How does this factor in time for appeals?

Each sentence gets its own line.
```suggestion
The Introductory Process may be held twice a semester at the discretion of the evaluations director.
If the evaluations director deems it necessary, the second intro process must begin within a week of the first process ending, and must last the same six (6) weeks.
```

Maybe this could be wrapped into the earlier portion of this paragraph, it seems a little bit out of place, and the `same six (6) weeks` is weird.

\asubsubsection{The Introductory Process}
The Introductory Process is designed to provide an easy means for Introductory Members to meet existing House members, learn House history, demonstrate their involvement potential to House, and allow existing House members to evaluate them for Active Membership.
\asubsubsubsection{The Evaluation Period}

> The Process will occur either once or twice per semester, and will last for six (6) weeks.

Reviewer 2: This allows for one intro process at the end of the semester... do we want that?

> The Process shall be initiated by the Evaluations Director during either the first or second week of each semester.

Reviewer 3: With this wording, the introductory process can only begin for one week, either the first or second week of the semester. Some additional wording to allow packets to start for the first two weeks of the Introductory Process would be good.

The Executive Board may hold a closed ballot Simple Majority vote to extend the Introductory Process or the Introductory Packet.
If an Introductory Process is extended, the Introductory Evaluation shall occur at the termination of the extended Process, which may be outside of the period defined in \ref{Introductory Evaluation}.

> If deemed necessary by the Evaluations Director, or by an Executive Board simple majority vote, the second intro process must begin within nine (9) days of the first process ending.

Reviewer 3: Some additional wording here to allow packets to start for the first two week of the second introductory process would be good

\asubsubsection{The Introductory Process}
The Introductory Process is designed to provide an easy means for Introductory Members to meet existing House members, learn House history, demonstrate their involvement potential to House, and allow existing House members to evaluate them for Active Membership.
\asubsubsubsection{The Evaluation Period}
The Process will occur either once or twice per semester, and will last for six (6) weeks.

> The Process shall be initiated by the Evaluations Director during either the first and second week of each semester.

Reviewer 3: Either-And is really weird wording-wise

Reviewer 3: Resolving this as the vote is now open and the PR cannot be changed. This can be fixed in a non-semantic change if the amendment passes.

\asubsubsubsection{Introductory Evaluation}
The Introductory Evaluation is the process by which House chooses which Introductory Members to extend an offer of full Resident or Non-Resident Membership.

> Occurs during the sixth week of each semester. If the evaluations director chooses to hold a second Introductory Process, the Introductory Evaluation will also occur during the twelfth week of each semester.

Reviewer: ```suggestion
Occurs at the conclusion of the sixth week of the Process.
```
Since the process may start the first or second week, maybe this should be tied to the length of the process, rather than the date?

Reviewer: Also, each sentence gets its own line
